### ISSUE TITLE
feature request: different zoom scales for horizontal/vertical

### ISSUE THREAD

**Issue by tpapp** [timholy/ProfileView.jl#165]
`@snoopi_deep` plots have a lot of tall & narrow regions, so zooming in then requires a lot of vertical panning.

I am not sure about the GUI for this, maybe Ctrl-Shift-scroll or something similar could just zoom horizontally.

**Comment by timholy** [comment]
Does Ctrl-click-drag zoom selection work for you? (It uses rubberband selection to pick the zoom region.)

**Comment by tpapp** [comment]
Yes, I can get what I want by that and then Ctrl-scrolling, but it would be more convenient to do it directly.

Just a thought, feel free to close.

**Comment by tpapp** [comment]
Just an explanation why I find the above finicky: zooming out (Ctrl-scroll) resets the aspect ratio to whatever fits in height when the plot fills the window vertically. Then, to go back, I need to rubberband carefully again.

**Comment by timholy** [comment]
OK, agreed. Re the interface: how about if you hold down the H key while scrolling to zoom just horizontally, and the V key for just vertical zoom?

**Comment by tpapp** [comment]
You mean Ctrl-V-scroll and Ctrl-H-scroll? That could work for guitar/piano players so I am OK with it :wink: Or just V-scroll and H-scroll? That would be perfect.
